.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: OLM Kubernetes Slack Channel
+    url: https://kubernetes.slack.com/archives/C0181L6JYQ2
+    about: Slack channel for OLM developers and the kubernetes community - good place to ask questions! 
```

Comment:
Maybe list the name of channel explicitly here would be nice: `Join #olm-dev Slack channel for OLM developers and the kubernetes community - good place to ask questions!`

Reply:
yep, good call